src/index.html

```diff
@@ -1,13 +1,38 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style/index.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap" rel="stylesheet">
+  </head>
+  <body>
+    <article class="card" data-qa="card">
+      <img
+        src="images/imac.jpeg"
+        alt="Моноблок APPLE A1419 iMac 27' Retina 5K (MNED2UAA)"
+        class="card__picture">
+      <h2 class="card__title">Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)</h2>
+      <p class="card__code">Код товара: 195434</p>
+      <div class="card__raiting">
+        <div class="stars stars--4">
+          <div class="stars__item"></div>
+          <div class="stars__item"></div>
+          <div class="stars__item"></div>
+          <div class="stars__item"></div>
+          <div class="stars__item"></div>
+        </div>
+        <div class="card__raiting-text"><p>Отзывов: 5</p></div>
```

Comment:
Semantic tags

Reply:
I used the tag <p>, or I shouldn`t use div?

Reply:
There is no need for an additional `<div>`, it's enough to use `<p>`